--- lib_pypy/_tkinter/tklib_build.py
+++ lib_pypy/_tkinter/tklib_build.py
@@ -22,12 +22,9 @@
     linklibs = ['tcl', 'tk']
     libdirs = []
 else:
-    for _ver in ['', '8.6', '8.5', '']:
-        incdirs = ['/usr/include/tcl' + _ver]
-        linklibs = ['tcl' + _ver, 'tk' + _ver]
-        libdirs = []
-        if os.path.isdir(incdirs[0]):
-            break
+    incdirs = []
+    linklibs = ['tcl8.6', 'tk8.6']
+    libdirs = []

 config_ffi = FFI()
 config_ffi.cdef("""
